### PR TITLE
Add applications support

### PR DESCRIPTION
**Pull request by vladhss**

### CODE REVIEW

**Comment by sahaqaa** [comment]
I will try to test it and get in touch with you with questions or proposals

**Comment by sahaqaa** [comment]
All manual tests passed OK.

**Review by arslanbekov — APPROVED** [review]

**Comment by arslanbekov** [comment]
```
Error: cloudconnexa/resource_application.go:253:27: appends: append with no values (govet)
  			groupedCst[protocol] = append(groupedCst[protocol])
  			                       ^
Error: issues found
```

File: cloudconnexa/resource_application.go [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+
+		mainConfig := configList[0].(map[string]interface{})
+		var cst = mainConfig["custom_service_types"].(*schema.Set)
+		var groupedCst = make(map[string][]cloudconnexa.Range)
+		for _, item := range cst.List() {
+			var cstItem = item.(map[string]interface{})
+			var protocol = cstItem["protocol"].(string)
+			var fromPort = cstItem["from_port"].(int)
+			var toPort = cstItem["to_port"].(int)
+			groupedCst[protocol] = append(groupedCst[protocol])
```
Comment (arslanbekov) [inline cloudconnexa/resource_application.go:253]:
```suggestion
```

Comment (arslanbekov) [inline cloudconnexa/resource_application.go:253]:
```suggestion
```

**Comment by sahaqaa** [comment]
@arslanbekov please advice - how you were able to find this? Which tool or command did you used?

> ```
> Error: cloudconnexa/resource_application.go:253:27: appends: append with no values (govet)
>   			groupedCst[protocol] = append(groupedCst[protocol])
>   			                       ^
> Error: issues found
> ```

Because "make" did build provider and was not complaining and usually Go complains when something is not used or not set.

**Comment by arslanbekov** [comment]
> how you were able to find this? Which tool or command did you used?

@sahaqaa of course, we [use](https://github.com/OpenVPN/terraform-provider-cloudconnexa/blob/main/.github/workflows/lint.yml#L25-L28) a [linter](https://github.com/golangci/golangci-lint-action) that allows us to lint the code.

You can run locally this linter:
1. ```go install github.com/golangci/golangci-lint/cmd/golangci-lint@latest```
2. ```golangci-lint run```

**Comment by sahaqaa** [comment]
Oh, i see, it found issue right away, thank you, will be using it in future :+1:

**Comment by vladhss** [comment]
@sahaqaa @arslanbekov Issue fixed, I replaced append with empty array initialization

**Review by arslanbekov — APPROVED** [review]

**Pull request merged**